examples/automm/tabular_dl/README.md

```diff
@@ -61,13 +61,35 @@ problem_type | regression | binary | multiclass | multiclass | binary | multicla
 #classes | - | 2 | 100 | 4 | 2 | 1000 | 2 | - | 7 | - | -
 Best in [1] | 0.459 | 0.859 | 0.396 | 0.732 | 0.729 | 0.963 | 0.8982 | 8.794 | 0.970 | 0.753 | 0.745
 FT-Transformer in [1] | 0.459 | 0.859 | 0.391 | 0.732 | 0.729 | 0.960 | 0.8982 | 8.855 | 0.970 | 0.756 | 0.746
-AutoMM FT-Transformer + Const Lr=1e-4 | 0.404 | 0.863 | 0.388 | 0.727 | 0.729 | 0.954 | OverflowError | 0.781 | 0.964 | AttributeError | 0.904
-AutoMM FT-Transformer + Cosine Lr=1e-4| 0.401 | 0.864 | 0.388 | 0.728 | 0.729 | 0.952 | OverflowError | 0.780 | 0.964 | 0.765 | 0.924
+AutoMM FT-Transformer | 0.482 | 0.859 |  |  |  | 0.949 | OverflowError |  | 0.963 |  | 0.761
 
-The trained weights and configs are accessible at https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/results.zip or s3://autogluon/results/tabular/results.zip. Use
+The `AutoMM FT-Transformer` column is runed by `run_all.sh`, where we use the following hyperparameter:
```

Comment:
You can reproduce the `AutoMM FT-Transformer` column by running `run_all.sh`, where we use the following hyperparameter:

Reply:
Great, thanks for the suggestion.